src/app/app.module.ts

```diff
@@ -1,13 +1,11 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
 import { LandingPageComponent } from './components/landing-page/landing-page.component';
 import { AppRoutingModule } from './app-routing.module';
 import { NavComponent } from './components/nav/nav.component';
 import { PlanetComponent } from './components/planet/planet.component';
-
 import { SpeciesListComponent } from './components/species-list/species-list.component';
-import { CharacterComponent } from './components/character/character.component';
+import { CharacterComponent } from "./components/character/character.component";
```

Comment:
All of the other imports use single quotes. Why change it now?

Reply:
None of us modified this import, so maybe that change came from master? Either way we corrected it on this branch.